### PR TITLE
Added Wildfly

### PR DESCRIPTION
**Pull request by phillip-kruger**
Hi.
I wanted to test webjars in wildfly swarm:

I did the following changes:

    Up the version to the latest.
    Removed jetty-runner
    Added maven profile to run with Wildfly swarm, using the wildfly maven plugin
    Externalize webjars version number and doing resource filtering.

### CODE REVIEW

File: nbactions.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jamesward) [inline nbactions.xml]: I think we can remove this file now.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,47 +1,57 @@
...
-            margin-top: 50px;
-        }
-    </style>
-    <script type="text/javascript">
-        $(function() {
-            $("#showModalButton").bind('click', function(event) {
-                $('#myModal').modal()
+    <head>
+        <title>WebJars Sample - Jetty WAR</title>
+        <link rel='stylesheet' href='webjars/bootstrap/${bootstrap.version}/css/bootstrap.min.css'>
```
Comment (jamesward) [inline src/main/webapp/index.html:5]: How do these values get populated?

File: nbactions.xml (hunk 1 above)
Comment (phillip-kruger) [reply nbactions.xml]: Yes sure, will do (sorry that is my IDE)

File: src/main/webapp/index.html (hunk 2 above)
Comment (phillip-kruger) [reply src/main/webapp/index.html:5]: This gets populated with maven build. It just makes it easier to up a version, you only need to up the version in the <properties> of maven pom.xml

<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-war-plugin</artifactId>
                <version>3.0.0</version>
                <configuration>
                    
                    <webResources>
                        <resource>
                            <directory>${basedir}/src/main/webapp</directory>
                            <filtering>true</filtering>
                            <includes>
                                <include>**/*.css</include>
                                <include>**/*.html</include>
                                <include>**/*.xhtml</include>
                            </includes>
                        </resource>
                    </webResources>
                    
                </configuration>
            </plugin>

**Pull request merged**